src/codec_aom.c

```diff
@@ -139,9 +139,9 @@ static avifBool aomCodecGetNextImage(avifCodec * codec, avifImage * image)
         if (image->profileFormat == AVIF_PROFILE_FORMAT_NONE) {
             // If the AVIF container doesn't provide a color profile, allow the AV1 OBU to provide one as a fallback
             avifNclxColorProfile nclx;
-            nclx.colourPrimaries = (uint16_t)codec->internal->image->cp;
-            nclx.transferCharacteristics = (uint16_t)codec->internal->image->tc;
-            nclx.matrixCoefficients = (uint16_t)codec->internal->image->mc;
+            nclx.colourPrimaries = (avifNclxColourPrimaries)codec->internal->image->cp;
+            nclx.transferCharacteristics = (avifNclxTransferCharacteristics)codec->internal->image->tc;
+            nclx.matrixCoefficients = (avifNclxMatrixCoefficients)codec->internal->image->mc;
```

Comment:
Joe: Because of the -Wno-conversion flag added in v0.7.0, we do not need any casts for these assignments. Please let me know if you would like me to simply remove the uint16_t casts. Thanks!

Reply:
I like the explicit casts (the way you changed it).

Reply:
Thanks. That's good to know. I will submit another pull request that allows us to build libavif with -Wconversion enabled, with the exception of the warning about isnan and isinf in cJSON.c due to a clang bug:

  https://bugs.llvm.org/show_bug.cgi?id=35268